x-pack/plugins/upgrade_assistant/README.md

```diff
@@ -0,0 +1,67 @@
+# Upgrade Assistant
+
+## About
+
+Upgrade Assistant helps users prepare their Stack for being upgraded to the next major. Its primary
+purposes are to:
+
+* **Surface deprecations.** Deprecations are features that are currently being used that will be
+removed in the next major. Surfacing tells the user that there's a problem preventing them
+from upgrading.
+* **Migrate from deprecation features to supported features.** This addresses the problem, clearing
+the path for the upgrade. Generally speaking, once all deprecations are addressed, the user can
+safely upgrade.
+
+### Deprecations
+
+There are two sources of deprecation information:
```

Comment:
Might be worth emphasising here that the UA only uses Deprecation Info API for surfacing index-level and cluster-level deprecations at the moment

Reply:
Good catch! Done.